src/components/SelectionList/BaseListItem.js

```diff
@@ -33,6 +34,18 @@ function BaseListItem({
     const isUserItem = lodashGet(item, 'icons.length', 0) > 0;
     const ListItem = isUserItem ? UserListItem : RadioListItem;
 
+    const rightHandSideComponentRender = () => {
+        if (canSelectMultiple || !rightHandSideComponent) {
+            return null; // no right hand side component
```

Comment:
i don't think this comment is needed. the code is self explanatory :)
```suggestion
            return null;
```

Reply:
Looks like another job of Captain Obvious!